src/components/drawer/drawer.js

```diff
@@ -75,7 +72,7 @@ export class DrawerComponent {
     this.querySelectors();
     this.initMoreBtns();
```

Comment:
you don't need more buttons in drawer anymore. they are in tags and amounts components

Reply:
I have not made changes in drawer in part of this.initMoreBtns();

Reply:
yes but you do not need this method anymore -- so delete it